src/main/java/org/vaadin/example/bookstore/ui/inventory/InventoryView.java

```diff
@@ -72,15 +71,17 @@ public HorizontalLayout createTopBar() {
         // null
```

Comment:
what is this null? probably should be removed.

Reply:
Done.